src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -180,7 +180,7 @@ public function __destruct()
             if (!$this->multi->openHandles) {
                 if ($this->logger) {
                     foreach ($this->multi->pushedResponses as $url => $response) {
-                        $this->logger->info(sprintf('Unused pushed response: %s', $url));
+                        $this->logger->warning(sprintf('Unused pushed response: "%s"', $url));
```

Comment:
info or debug: if you don't care for the pushes, this shouldn't spam the logs

Reply:
This is wrong

Reply:
why is it wrong? no reading the css/js pushed while crawling an html resource is not wrong